### PR TITLE
added react_week1 folder(todo_app)

### PR DESCRIPTION
**Pull request by soujanyapolavarapu**

### CODE REVIEW

File: react_week1/src/App.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+},
+{
+  description: 'Eat breakfast',
+  deadLine: 'Fri Sep 15 2017'
+}
+];
+
+function App() {
+  return (
+    <div className="todo-App">
```
Comment (khjcph) [inline react_week1/src/App.js:22]: it is a little more idiomatic to use "todo-app" here. Attributes like these are generally case insensitive.

File: react_week1/src/App.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+<RenderTodo toDoList = {toDoList}/>
```
Comment (khjcph) [inline react_week1/src/App.js:23]: try to work a bit with the indentation here.

File: react_week1/src/todo.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        <ol>
+            <h1 className = 'todo-header'>
+               Todo List 
+            </h1>
+        { 
+    props.toDoList.map(todo => <li><Todo description = {todo.description} deadLine = {todo.deadLine}/> </li>)
+    } 
+        </ol>
+    );
+
```
Comment (khjcph) [inline react_week1/src/todo.js:25]: maybe work a little with the formatting here. for example:

```
return (
  <ol>
    <h1 className="todo-header">Todo List</h1>
    {props.toDoList.map((todo) => (
      <li>
        <Todo description={todo.description} deadLine={todo.deadLine} />{" "}
      </li>
    ))}
  </ol>
);
```

is a little easier to read.

File: react_week1/src/todo.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const Todo = (props) =>{
```
Comment (khjcph) [inline react_week1/src/todo.js:3]: When passing props, you can actually use a very cool javascript feature called destructuring. For example

    const Todo = ({description, deadline}) => { }

Then you can access the variables without using `props.`

**Review by khjcph — APPROVED** [review]
Great work!! 👍

I left a few comments that you can look at if you want. One of them is about formatting. This is much easier if using a formatter / linter inside your code editor. I hope we will get to cover that in class.

I don't remember if we covered destructuring in class. If not, you can just ignore that.

File: react_week1/src/todo.js (hunk 4 above)
Comment (soujanyapolavarapu) [reply react_week1/src/todo.js:3]: Hi @kasper, Thank you for your all valuable comments.I will rectify as soon as possible. :)

**Pull request merged**